department-of-veterans-affairs/caseflow
# Add a VHA Program Office section to the team management page

### Description
Puts VHA Program Offices into their own section on the Team Management page.

### Acceptance Criteria

### Testing Plan
1. Run VHA Program Office DB seed: https://github.com/department-of-veterans-affairs/caseflow/pull/16646
2. Go to `/team_management`
3. Confirm VHA Program Offices appear in their own section.

 |BEFORE|AFTER|
 |---|---|
 |<img width="1773" alt="Screen Shot 2021-08-19 at 8 45 53 AM" src="https://user-images.githubusercontent.com/640182/130071640-7e0c0b0f-39b5-45fb-aa81-b21a8553309e.png">|<img width="1775" alt="Screen Shot 2021-08-19 at 8 45 19 AM" src="https://user-images.githubusercontent.com/640182/130071668-ff76ef3d-9b02-4c84-b642-455bf08e12b0.png">|
